src/components/ont/OntFlowcell.vue

```diff
@@ -1,10 +1,17 @@
 <template>
-  <g :transform="getMatrix">
-    <rect width="61" height="227"/>
+  <g :transform="getMatrix" v-on:drop="drop" v-on:dragover="allowDrop" >
+    <text x="25" y="30" class="medium">{{ position }}</text>
+
+    <rect width="61" height="227" v-bind:class="status"/>
+
+    <foreignObject y="100" width="70" height="227">
+      <b-form-input v-model="libraryName" placeholder="Name" :id="'libraryNameInput-'+this.position" @change="updateFlowcell"></b-form-input>
```

Comment:
It is difficult to see the full name of the library when it has been dropped onto the flowcell.

Reply:
Made it a bit wierd but struggled with SVG HTML/CSS -> Added a tooltip as a quick fix

Reply:
ok. Lets add this issue to #421 . We can do a deep dive into the CSS to see what is going on. If we get some time next week we can have more of a look. I know how much of a pain CSS can be especially when it is wrapped up in bootstrap.